modules/calypsoify/class.jetpack-calypsoify.php

```diff
@@ -295,15 +295,52 @@ private function installed_plugins_icon() {
 		return 'data:image/svg+xml;base64,' . base64_encode( $svg );
 	}
 
-	public function get_close_gutenberg_url() {
+	function get_calypso_origin() {
```

Comment:
Would you mind adding method visibility declarations and method docs?

Reply:
Added in c995d6c.